src/Utils.php

```diff
@@ -744,7 +744,8 @@ public static function check_products_for_letterbox( $products ) {
 
 			// If one of the item is not letterbox product, then the order is not eligible automatic letterbox.
 			// Thus should return false immediately.
-			if ( 'yes' !== $is_letterbox_product ) {
+			$is_digital = $product->is_virtual() || $product->is_downloadable();
```

Comment:
I suggest to check if its virtual product before getting `Product\Single::LETTERBOX_PARCEL` meta, if its virtual there is no need to get the meta value.

Reply:
yes you are right